0xProject/0x-api
# feat: Publish Quote Reports to Kafka

# Description

Allow Quote reports to be published to Kafka when including `KAFKA_BROKERS` is set. The Quote Reports will still be logged as usual.

# Testing Instructions

1. Setup https://github.com/0xProject/events_approval-parser
2. Configure `KAFKA_BROKERS` to point to the Kafka instance configured above
3. Querying the API should produce events (only for firm quotes)

# Checklist



    For more information see `0x API Matcha smoke test runbook` in Quip.
